test/svgp.jl

```diff
@@ -0,0 +1,4 @@
+@testset "svgp" begin
```

Comment:
Could be removed now that there's actual tests being run as well

Reply:
Yes, sorry that's still WIP - I need to look through https://github.com/JuliaGaussianProcesses/AbstractGPs.jl/pull/179 and see what I can use from that here

Reply:
I've added the standard AbstractGP interface tests